docs/release-notes/highlights-7.6.0.asciidoc

```diff
@@ -0,0 +1,69 @@
+[[release-highlights-7.6.0]]
+== 7.6.0 release highlights
+++++
+<titleabbrev>7.6.0</titleabbrev>
+++++
+
+//NOTE: The notable-highlights tagged regions are re-used in the
+//Installation and Upgrade Guide
+
+// tag::notable-highlights[]
+
+Explore the new features in Kibana 7.6.
+
+[cols="50, 50"]
+|===
+
+a| *Elastic&nbsp;Maps improvements*
+
+Color the data points on your map based on the discrete values in a categorical field.
+Add text labels to give your viewers more information about the data you’re
+overlaying on your maps.
+
+| image:release-notes/images/7-6-maps-category.png[Categorical styling in Maps]
+
+a| *Embeddable maps in Canvas*
+
+Incorporate a geospatial perspective into a Canvas workpad.
+
+| image:release-notes/images/7-6-canvas-map.png[Embedded maps in Canvas]
+
+
+a| *Lens improvements*
+
+Use scripted fields in your visualizations, just like any other field in the index.
+Rapidly reset a layer
+with a single click instead of removing data fields one at a time.
+
+| image:release-notes/images/7-6-lens-reset-layer.png[Scripted fields in Lens]
+
+a| *Nested field support*
+
+Query on nested fields using KQL.
+You can query for a user whose first name is Tom and whose
+last name is Hanks, or a user whose first name is Tom and last name is Smith.
+<<kuery-query-nested-field, Our documentation>> contains examples for you to follow.
```

Comment:
This content fails when it's pulled into the Installation/Upgrade Guide unless you make it an external link:
```suggestion
{kibana-ref}/kuery-query-nested-field.html[Our documentation] contains examples for you to follow.
```